app/views/articles/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<p style="color: green"><%= notice %></p>
+<p>Article #<%= @article.id %> <p>
+
+<%= @article.title %>
+<%= @article.content %>
```

Comment:
How would you feel about the following? We should also move the strings into the locale file the same way as the other strings. This should make the code more readable and maintainable.

```suggestion
<p><strong>Title:</strong><%= @article.title %></p>

<p><strong>Content:</strong><%= @article.content %></p>
```